### ISSUE TITLE
BOINC not releasing resources when suspended.

### ISSUE THREAD

**Issue by Nichotj** [BOINC/boinc#5901]
When BOINC is suspended, there are a number of activities still running which are consuming CPU and memory.

prst-v10i.exe is consuming CPU and hsgamma_FGRP5_1.08_windows_intelx86__FGRPSSE.exe is consuming memory

![Image](https://github.com/user-attachments/assets/c3392a90-ddeb-46e4-b556-30de73bdae40)
![Image](https://github.com/user-attachments/assets/4dedbaf4-d24b-40a0-ac64-40914f0199f7)

When computing is suspended, BOINC shouldn't be using CPU, this time there are only 2 running, but I've seen 5 running which really slowed the laptop down.

**System Information**
 - OS: Win 11 Pro
 - BOINC Version: 8.0.2

Thank you

**Comment by Nichotj** [comment]
@AenBleidd I do, but I haven't changed these settings.  But there was a new version recently.
I'll try turning it off then.

Thanks

**Comment by Nichotj** [comment]
@AenBleidd That might explain the memory usage, but would that explain the high cpu activity when suspended?

Thanks

**Comment by AenBleidd** [comment]
> but would that explain the high cpu activity when suspended?

No, this looks like an issue. However, it's not clear whether this is an issue of BOINC client or of the Project's application.
How often to you have this issue?
Does it happen with one particular application or with several applications? If the latter is yes, are all these applications from one project or from the different projects?

**Comment by Nichotj** [comment]
Ok, so turning that field off seems to have cured both issues for me.
It was happening fairly consistently for me, but I didn't look at what projects/tasks were running at the time.
Is there something that I can provide that will answer your questions?

Thanks.

**Comment by kotenok2000** [comment]
If you have page file enabled windows offloads suspended program memory there when something else needs more memory.
